frontend/src/shared/helpers.tsx

```diff
@@ -233,7 +233,6 @@ export const SampleDetailsColumns: ColDef<SampleMetadataExtended>[] = [
   {
     field: "primaryId",
     headerName: "Primary ID",
-    editable: (params) => !protectedFields.includes(params.colDef.field!),
```

Comment:
These individual `editable` properties in each column definition are redundant as we already define them [here](https://github.com/mskcc/smile-dashboard/pull/107/files#diff-db966dd79103f32fddb49dcefcfba78d4325506ce364cd0e24779a3521aa7173R449).